src/traversal/diffRelayQuery.js

```diff
@@ -11,6 +11,8 @@
  * @typechecks
  */
 
+/* eslint-disable consistent-return */
```

Comment:
Thoughts on just fixing the 1 instance instead? I'm not completely sold that it's the most valuable rule but I'd like to make that decision on a broader level and try to avoid file-scope disables.